src/v1/sources/cordial/mapping.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "sourceKeys": "event._id",
+    "destKeys": "properties.event_id"
+  },
+  {
+    "sourceKeys": "contact._id",
+    "destKeys": ["context.traits.userId", "userId"]
+  },
+  {
+    "sourceKeys": ["event.ts", "event.ats"],
+    "destKeys": ["timestamp", "sentAt", "originalTimestamp"]
+  },
+  {
+    "sourceKeys": "event.d",
+    "destKeys": "context.device"
```

Comment:
What is stucture of `d` `device` is expected to object check here https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/

Reply:
This d is giving the details of device. that's why I have mapped it.

Reply:
Can you once share the structure is it an object?

Reply:
`d: {
                    type: 'computer',
                    device: 'Macintosh',
                    platform: 'OS X',
                    browser: 'Chrome',
                    robot: false,
                  },`